gcam-build.md

```diff
@@ -133,11 +133,11 @@ Note that even if you turn off Java support you can still have GCAM generate the
 Which can subsequently be loaded into an XML database by using the [Model Interface](user-guide.html#modelinterface) by opening a database, choosing `File -> Manage DB`, then Click `Add`, finally select the `debug_db.xml` document to add to the database.  Note a _new_ database can be created by simply selecting an empty folder to open as a database (you will see a warning message about potentially deleting files **and you should pay attention to it**).
 
 #### 2.3.2 Java On Windows
-On Windows users can get by with just installing the Standard Runtime Environment (JRE) for running GCAM however when building the Java Development Kit (JDK) is recommended.  GCAM will expect the following header and lib files within `<GCAM Workspace>/libs`:
+On Windows users can get by with just installing the Standard Runtime Environment (JRE) for running GCAM however when building the Java Development Kit (JDK) is recommended.  Note the openJDK only provides a zip archive and does not come with a windows installer, which isn't strictly necessary for it to work with GCAM however users may find the version distributed by [Redhat](https://developers.redhat.com/products/openjdk/download) easier because it does provide a Windows installer.  GCAM will expect the following header and lib files within `<GCAM Workspace>/libs`:
```

Comment:
Does this "running GCAM however when building the Java Development Kit (JDK) is recommended" mean:

"running GCAM, however when building GCAM we recommend using the Java Development Kit (JDK)."

Reply:
It is actually only if you need to re-compile the Java parts, not even when compiling the C++.  Although that being said it is confusing and I could be convinced to remove it and just recommend installing the openJDK development kit.

Reply:
So if most of this only about development (in which case it is probably useful to document), would this work?:

On Windows users can get by with just installing the Standard Runtime Environment (JRE) for running GCAM. 

For development of the Java components of GCAM on windows systems (which is likely not necessary for most users), the open Java Development Kit (JDK) is recommended.  Note the openJDK only provides a zip archive and does not come with a windows installer, which isn't strictly necessary for it to work with GCAM however users may find the version distributed by [Redhat](https://developers.redhat.com/products/openjdk/download) easier because it does provide a Windows installer.  GCAM will expect the following header and lib files within `<GCAM Workspace>/libs`: